cleanrepo/Program.cs

```diff
@@ -595,16 +595,21 @@ private static void ListOrphanedIncludes(string inputDirectory, Dictionary<strin
         DirectoryInfo? rootDirectory = null;
 
         // Get all files that could possibly link to the include files
-        List<FileInfo>? files = HelperMethods.GetAllReferencingFiles("*.md", inputDirectory, ref rootDirectory);
-
-        if (files is null || rootDirectory is null)
+        List<FileInfo>? mdfiles = HelperMethods.GetAllReferencingFiles("*.md", inputDirectory, ref rootDirectory);
+        List<FileInfo>? ymlfiles = HelperMethods.GetAllReferencingFiles("*.yml", inputDirectory, ref rootDirectory);
+        
+        if ((mdfiles is null && ymlfiles is null) || rootDirectory is null)
             return;
 
+        List<FileInfo> allReferencingFiles = mdfiles ?? [];
+        if (ymlfiles is not null)
+            allReferencingFiles.AddRange(ymlfiles);
+
         // Gather up all the include references and increment the count for that include file in the Dictionary.
         //foreach (var markdownFile in files)
-        Parallel.ForEach(files, markdownFile =>
+        Parallel.ForEach(allReferencingFiles, referencingFile =>
```

Comment:
@gewarren Do you like these for the renamed loop files? `allReferencingFiles` from your change, so a single file is now a `referencingFile` singular?